src/components/chatMenuDropdown/chatMenuDropdown.styles.scss

```
@@ -0,0 +1,173 @@
+		height: 20px;
+		flex-shrink: 0;
+		color: $form-secondary;
+		margin-top: 2px;
+
+		svg {
+			width: 20px;
+			height: 20px;
+		}
+
+		svg * {
+			fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)